lib/fastlane/plugin/create_remote_config_release_package/actions/create_remote_config_release_package_action.rb

```diff
@@ -46,15 +47,16 @@ def self.run(params)
           end
         end
 
-        versionNumber = other_action.get_version_number.gsub(/[^0-9]/, '').ljust(7, '0')
+        marketingVersion = other_action.get_version_number(xcodeproj: projectFile, target: targetName)
```

Comment:
Is it possible to add an if statement that checks if the targetName has been provided, then use it otherwise default to the initial implementation.

Reply:
Done!